G/GAP_pkg/GAP_pkg_semigroups/build_tarballs.jl

```diff
@@ -50,6 +50,9 @@ dependencies = gap_pkg_dependencies(gap_version)
 platforms = gap_platforms()
 platforms = expand_cxxstring_abis(platforms)
 
+# TODO: re-enable the below platforms
+filter!(p -> arch(p) != "riscv64", platforms)
```

Comment:
What is the failure for semigroups?

Reply:
```
[14:22:57] src/froidure-pin-max-plus-mat.cpp:41:1: fatal error: error writing to /tmp/ccIbjhhI.s: No space left on device
[14:22:57]    41 | }
[14:22:57]       | ^
[14:22:57] compilation terminated.
```
could also be a fluke in the filesystem, but I don't wanna rebuild all the other things here over and over again, so this will be further examined in a new PR